app/src/main/res/values/localpages.xml

```diff
@@ -6,21 +6,21 @@
     <!-- IMPORTANT NOTE: If this file is updated anyhow, the STRs wiki must be updated accordingly:
     https://github.com/MozillaReality/FirefoxReality/wiki/L10n -->
 
-    <!-- This string is used as the title of the Private Browsing page that is opened when the
+	  <!-- This string is used as the title of the Private Browsing page that is opened when the
     Private Browsing Mode button from the tray is clicked. -->
-    <string name="private_browsing_title">Private Browsing</string>
+    <string name="private_browsing_title">隱私瀏覽模式</string>
```

Comment:
oops, this file should not be changed (!)

Reply:
I'm not sure I understand?
If you mean that "Private Browsing" shouldn't be translated: we translate it for all other products (but not sure you're talking about that?)